jniwrapper/java/src/main/java/org/jitsi_modified/sctp4j/SctpJni.java

```diff
@@ -168,4 +172,31 @@ public static int onSctpOutboundPacket(
         // FIXME handle tos and set_df
         return -1;
     }
+
+    /**
+     * Method fired by native counterpart to log a debug message.
+     * @param message the message to log
+     */
+    public static void logDebug(String message) {
+        message = message.trim();
```

Comment:
The trim() is just to remove the "\n" that the native code adds? Can we just remove the "\n" from the native code?

Reply:
At least for logError, I have a fallback to native (v)printf if we can't successfully call the Java method, so you can get meaningful error messages for failure to load the native classes.

Also the debug function is passed as a native callback to the usrsctp library, and I assume that has newlines.